spyder/widgets/variableexplorer/tests/test_collectioneditor.py

```diff
@@ -265,5 +269,14 @@ def test_edit_mutable_and_immutable_types(qtbot, monkeypatch):
                                                             readonly=True)
 
 
+def test_view_module_in_coledit(qtbot, monkeypatch):
+    """Checks that modules don't produce an error when trying to open them in
```

Comment:
Please change `Checks` to `Check`

Reply:
Oops; fixed the one above but missed that one. Thanks for the catch; corrected now.